src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -340,10 +340,14 @@ public async Task<HttpResponseMessage> HandleRequestAsync(HttpRequestMessage req
             var createdTimeFrom = GetDateTimeQueryParameterValue(queryNameValuePairs, CreatedTimeFromParameter, default(DateTime));
             var createdTimeTo = GetDateTimeQueryParameterValue(queryNameValuePairs, CreatedTimeToParameter, default(DateTime));
             var runtimeStatus = GetIEnumerableQueryParameterValue<OrchestrationRuntimeStatus>(queryNameValuePairs, RuntimeStatusParameter);
-            var pageSize = GetIntQueryParameterValue(queryNameValuePairs, PageSizeParameter);
+
+            if (!TryGetIntQueryParameterValue(queryNameValuePairs, PageSizeParameter, out int pageSize))
+            {
+                pageSize = 100;

```

Comment:
This is the reason your tests are failing. The previous behavior assigned this to 0. We can change the default to 100, but if you notice below, it's calling a different GetStatusAsync. We'll have to fix the unit tests. I intentionally left this change out because I didn't want to hold up the release.

Reply:
Yep you are right. My next commit fixed one of the tests to do the right thing. The other tests still need to be modified, however.